lib/tasks/db.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+if Rake::Task['db:schema:dump:clickhouse'].present?
+  Rake::Task['db:schema:dump:clickhouse'].enhance do
+    Rake::Task['db:clickhouse:filter'].invoke
+  end
+end
+
+namespace :db do |args|
+  desc 'Filter secrets from clickhouse schema file'
+  task 'clickhouse:filter' do
+    migration_file = 'db/clickhouse_schema.rb'
+    text = File.read(migration_file)
+    new_contents = text.gsub(ENV['LAGO_KAFKA_BOOTSTRAP_SERVERS'], '*****')
+    File.open(migration_file, "w") {|file| file.puts new_contents }
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)